packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.spec.tsx

```diff
@@ -604,5 +604,34 @@ describe("horizontal overflow", (): void => {
 
         (rafThrottle as any) = actualRafThrottle;
     });
+    test("getPositionData returns correct values as scrollLeft value changes", (): void => {
```

Comment:
i would break this test out to have:
1. a test to check you are at the beginning
2. a test to check you are at the end
3. a test to check you are in the middle

Reply:
Sure